front/src/components/header/subcomponents/LoginModal.tsx

```diff
@@ -40,6 +41,7 @@ type Props = OwnProps & DispatchProps;
 type FormData = {
   email: string;
   password: string;
+  otp: string;
```

Comment:
Can we change all instances to `oneTimeCode`? Including the API endpoint please {API.TV_MINDER}/oneTimeCodeVerification